### ISSUE TITLE
rpn_loc_loss = nan

### ISSUE THREAD

**Issue by leetesua** [DetectionTeamUCAS/Faster-RCNN_Tensorflow#6]
![image](https://user-images.githubusercontent.com/25045281/42670458-9699f022-868d-11e8-8663-430d66859193.png)

hi, guys, when i train with default VOC dataset (and i did not change the version, num_class or add any dataset_name, just keep it default), I got this: rpn loss equals nan. Any idea about why this happens? Thanks a lot!

**Comment by leetesua** [comment]
Yes, the model did a good job when running the demo. @yangxue0827

**Comment by xtanitfy** [comment]
I have trained with this problems too.

**Comment by xtanitfy** [comment]
I think the reason i here:
____________________________________________________________
var_in_graph:  Fast-RCNN/resnet_v1_50/block4/unit_3/bottleneck_v1/conv3/BatchNorm/gamma:0
var_in_ckpt:  resnet_v1_50/block4/unit_3/bottleneck_v1/conv3/BatchNorm/gamma
____________________________________________________________
var_in_graph:  Fast-RCNN/resnet_v1_50/block4/unit_3/bottleneck_v1/conv3/BatchNorm/beta:0
var_in_ckpt:  resnet_v1_50/block4/unit_3/bottleneck_v1/conv3/BatchNorm/beta
____________________________________________________________
var_in_graph:  Fast-RCNN/resnet_v1_50/block4/unit_3/bottleneck_v1/conv3/BatchNorm/moving_mean:0
var_in_ckpt:  resnet_v1_50/block4/unit_3/bottleneck_v1/conv3/BatchNorm/moving_mean
____________________________________________________________
var_in_graph:  Fast-RCNN/resnet_v1_50/block4/unit_3/bottleneck_v1/conv3/BatchNorm/moving_variance:0
var_in_ckpt:  resnet_v1_50/block4/unit_3/bottleneck_v1/conv3/BatchNorm/moving_variance
____________________________________________________________

**Comment by xtanitfy** [comment]
The reason is the file of 'data/io/convert_data_to_tfrecord.py' change the code as follow:
========================================
if child_of_root.tag == 'object':
            label = None
            for child_item in child_of_root:
                if child_item.tag == 'name':
                    label = NAME_LABEL_MAP[child_item.text]
                if child_item.tag == 'bndbox':
                    tmp_box = []
                    for node in child_item:
                        if node.tag == 'xmin':
                            xmin = int(node.text)
                        if node.tag == 'ymin':
                            ymin = int(node.text)
                        if node.tag == 'xmax':
                            xmax = int(node.text)
                        if node.tag == 'ymax':
                            ymax = int(node.text)
                    tmp_box = [xmin,ymin,xmax,ymax]      
                    #tmp_box.append()
                    assert label is not None, 'label is none, error'
                    tmp_box.append(label)
                    box_list.append(tmp_box)
===================================
